conf/deployment/aws/ipi_3az_rhcos_3m_3w_3i.yaml

```diff
@@ -0,0 +1,27 @@
+---
+DEPLOYMENT:
+  allow_lower_instance_requirements: false
+  ocs_operator_nodes_to_label: 0
+ENV_DATA:
+  platform: 'aws'
+  deployment_type: 'ipi'
+  region: 'us-east-2'
+  worker_availability_zones:
+    - 'us-east-2a'
+    - 'us-east-2b'
+    - 'us-east-2c'
+  infra_availability_zones:
+    - 'us-east-2a'
+    - 'us-east-2b'
+    - 'us-east-2c'
+  master_availability_zones:
+    - 'us-east-2a'
+    - 'us-east-2b'
+    - 'us-east-2c'
+  worker_replicas: 3
+  master_replicas: 3
+  infra_replicas: 3
+  worker_instance_type: 'm5.4xlarge'
+REPORTING:
+  polarion:
+    deployment_id: 'OCS-xxxx'
```

Comment:
Is there an actual polarion ID that should be here?  If not let's remove the whole reporting section from this file.

Reply:
I will get try to get this added to polarion, if not I will remove this.

Reply:
done